atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SerializableTransaction.java

```diff
@@ -865,6 +827,66 @@ private void handleTransactionConflict(TableReference tableRef) {
                 tableRef, getTimestamp(), System.currentTimeMillis() - timeCreated);
     }
 
+    private BatchingVisitable<Map.Entry<Cell, byte[]>> wrapWithColumnRangeChecking(
```

Comment:
[Sanity check]: This code is semantically the same as the previous code, right? As far as I can tell, the main change in this class is the use of KeyedStream over Maps.transformEntries?

Reply:
Yep, that's right.